util/cron/test-valgrind.bash

```diff
@@ -12,4 +12,4 @@ unset CHPL_TEST_LIMIT_RUNNING_EXECUTABLES
 
 export CHPL_NIGHTLY_TEST_CONFIG_NAME="valgrind"
 
-$CWD/nightly -cron ${nightly_args} $(get_nightly_paratest_args 20)
+$CWD/nightly -cron -valgrind -examples ${nightly_args} $(get_nightly_paratest_args 20)
```

Comment:
`-nodepara 20 ` is probably excessive for examples only. I'd drop to 5-6

Reply:
Done.